src/LegendList.tsx

```diff
@@ -460,19 +489,6 @@ const LegendListInner: <T>(props: LegendListProps<T> & { ref?: ForwardedRef<Scro
             [],
         );
 
-        useEffect(() => {
```

Comment:
@jmeistrich Because we pass initialContentOffset to the refState.current during initialisation, this useEffect should not longer be needed.